pkgs/by-name/lu/lunapnr/package.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, fetchFromGitHub
+, fetchFromBitbucket
+, gcc11Stdenv
+
+  # dependencies
+, boost
+, cmake
+, eigen
+, ninja
+, opensta
+, python3
+, qt5
+}:
+
+let
+  version = "0.1.6";
+
+  tinysvgpp = fetchFromBitbucket {
+    owner = "trcwm";
+    repo = "tinysvgpp";
+    rev = "2589ed";
+    hash = "sha256-VmD1NJUPUET1LMHn6IPTLaSJ0HEk/Q158xT0+3R9LFc=";
+  };
```

Comment:
This exists as a submodule within the LunaPnR repo, but even with `fetchSubmodules = true`, it is not being fetched (maybe because of different vendors?)